test/live_view_native/components_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule LiveViewNative.ComponentsTest do
   import Meeseeks.CSS
 
   test "test_component/1 renders as expected" do
-    web_context = LiveViewNativePlatform.Kit.compile(%LiveViewNative.Platforms.Web{})
+    web_context = LiveViewNativePlatform.Kit.compile(%LiveViewNative.Platforms.HTML{})
     test_context = LiveViewNativePlatform.Kit.compile(%LiveViewNative.TestPlatform{})
     web_result =
       render_component(&TestComponents.test_component/1, platform_id: :web, native: web_context)
```

Comment:
you should also look out for things like `web_context` and change over to `html_context`

Reply:
also note the `web_result` variable on LOC 11

Reply:
Good call @bcardarella - and that actually shows that @supernintendo missed a `platform_id: :web` change to `:html` in the `components_test.exs` [LOC 12](https://github.com/liveview-native/live_view_native/pull/58/files/e9bbc8a54ccd62d8ee97bd2da9ad0c2cc16cdc2a#diff-41d3f84688c000c3b01b84af12912d60ab7efff823125f9838b0d9725550fbb4R12) that I fixed.